elytron/src/main/java/org/wildfly/extension/elytron/AuthenticationClientDefinitions.java

```diff
@@ -141,8 +141,7 @@
             .setFlags(AttributeAccess.Flag.RESTART_RESOURCE_SERVICES)
             .build();
 
-    static final PropertiesAttributeDefinition MECHANISM_PROPERTIES = new PropertiesAttributeDefinition.Builder(ElytronDescriptionConstants.PROPERTIES, true)
-            .setName(ElytronDescriptionConstants.MECHANISM_PROPERTIES)
+    static final PropertiesAttributeDefinition MECHANISM_PROPERTIES = new PropertiesAttributeDefinition.Builder(ElytronDescriptionConstants.MECHANISM_PROPERTIES, true)
```

Comment:
I am not sure about this change, is this a case of a referenced attribute loosing it's name anyway otherwise this looks like a management model change to rename the attribute.